content/doc/book/managing/cli.adoc

```diff
@@ -242,6 +242,26 @@ You can also precede the argument with `@` to load the same content from a file:
 java -jar jenkins-cli.jar [-s JENKINS_URL] -auth @/home/kohsuke/.jenkins-cli command ...
 ----
 
+The third and last authentication method is to configure environment variables in similar way as the `$JENKINS_URL`
+is used. The `username` can be specified via the environment variable `$JENKINS_USER_ID` while the `apitoken` can
+be specified via the variable `$JENKINS_API_TOKEN`. Both variables have to be set all at once.
+
+In case these environment variables are configured you could override the authentication method using different 
+credentials with the `-auth` option, which takes preference over them.
+
+[source]
+----
+export JENKINS_USER_ID=kohsuke
+export JENKINS_API_TOKEN=abc1234ffe4a
+java -jar jenkins-cli.jar [-s JENKINS_URL] command ...
+----
+
+[WARNING]
+====
+For security reasons the recommended way to load the authentication credentials is the use of a file 
```

Comment:
I'm not sure this is clear. I understand you mean specifying the name of the file with @ before but I think it's not easy to understand that if you don't know the mechanism beforehand.